azure-pipelines/publishing/github-release.yml

```diff
@@ -9,96 +9,95 @@ pr:
 pool: dotestlab
 
 variables:
```

Comment:
>variables [](start = 0, length = 9)

Can we add a variable to allow us to test the pipeline without actually performing a release? Like a dry run that does everything except the last stage. Instead of the last stage, maybe we can just recursively list everything available in the artifact directory as the final step.

Reply:
Yeah I've added it as a parameter so the pipeline UI reflects it in a toggle checkbox

---
In reply to: [585962244](https://github.com/microsoft/do-client/pull/7#discussion_r585962244) [](ancestors = 585962244)